games-action/openjazz/openjazz-20180522.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A free, open-source version of classic Jazz Jackrabbit games"
+DESCRIPTION="A free, open-source version of classic Jazz Jackrabbit games.\
+Requires the original game data to work.\
+Please put the data into the folder:\
+~/config/non-packaged/Data/openjazz"
+HOMEPAGE="https://github.com/AlisterT/openjazz"
+COPYRIGHT="Alister Thomson"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/AlisterT/openjazz/releases/download/$portVersion/openjazz-$portVersion.tar.xz"
+CHECKSUM_SHA256="35260066f307ff7e31304907a2a430a20b8d1fa5f90ad2c13275ab3aca337c24"
+SOURCE_DIR="openjazz-$portVersion"
+ADDITIONAL_FILES="openjazz.rdef"
+PATCHES="openjazz-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	openjazz = $portVersion
+	app:OpenJazz = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+#	lib:libxmp
+	lib:libmodplug
+	lib:libz
+    "
+
+BUILD_REQUIRES="
+    haiku_devel
+	devel:libsdl
+#	devel:libxmp
+	devel:libmodplug
+	devel:libz
+    "
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
+	cmd:gcc
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	export LDFLAGS="-lnetwork -lbe"
```

Comment:
`$LDFLAGS` can be omitted when I merge the [buildsystem branch](https://github.com/AlisterT/openjazz/blob/buildsystem/configure.ac#L17) and do the next release.